bower.json

```diff
@@ -10,7 +10,7 @@
     "easteregg": "https://github.com/mozilla/fxa-easter-egg.git#ab20cd517cf8ae9feee115e48745189d28e13bc3",
     "fxa-checkbox": "mozilla/fxa-checkbox#7f856afffd394a144f718e28e6fb79092d6ccddd",
     "fxa-content-server-l10n": "https://github.com/mozilla/fxa-content-server-l10n.git",
-    "fxa-js-client": "https://github.com/mozilla/fxa-js-client.git#0.1.48",
+    "fxa-js-client": "https://github.com/mozilla/fxa-js-client.git#0.1.49",
```

Comment:
Fwiw, this is really the most important part of the change. I'd like to ship this so we can delete all the duplicated validation/processing logic from the auth server.